common/story_cycles/story_cycle_pet_dog.txt

```diff
@@ -1,19 +1,26 @@
 ﻿story_cycle_pet_dog = {
 
+	#Warcraft
 	on_setup = {
 		assign_dog_gender_effect = { GENDER = random }
-		assign_dog_fur_color_effect = { COLOR = random }
 		assign_dog_eye_color_effect = { COLOR = random }
+		assign_companion_lifespan_effect = yes
+		if = {
+			limit = {
+				story_owner = {
+					has_furred_companion_trigger = yes
+				}
+			}
+			assign_dog_fur_color_effect = { COLOR = random } #All companions should have this, change colors depending on the species
+		}
```

Comment:
You say all companions should have this (and this most likely true because it's used in texts), but at the same time you block it behind the trigger.

Reply:
I added that comment as a TODO, I haven't done it yet.

Reply:
Done in dcfbd28